lib/services/database_service.dart

```diff
@@ -225,10 +276,15 @@ class DatabaseService {
       _firestoreSubscriptions.profilePics =
           _firestore.connectToProfilePics(userId, _storeController);
     } catch (error, trace) {
-      _storeController.add(AddProblem.from(
-          message: error.toString(),
-          traceString: trace.toString(),
-          type: ProblemType.observeProfilePics));
+      _storeController.add(
+        AddProblem(
+          problem: ObserveProfilePicsProblem.by(
```

Comment:
Yeah this seems more natural to me - an "ObserveProfilePicsProblem" being created on error here. The question of whether errors occuring inside the "connect..." extension functions should have different names, I'm not sure what my opinion is yet. I'm wondering if we might not deal with them differently on a separate PR. Basically, feel free to ignore me :-)

Reply:
Circling back: this will presumably be part of the [Can we reduce boilerplate at the point of catching errors?](https://app.asana.com/0/0/1199189031575780) Conversation.